src/index.html

```diff
@@ -8,5 +8,38 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    <ul class="calendar calendar--start-day_sun calendar--length_31">
```

Comment:
```suggestion
    <ul class="calendar calendar--sun calendar--31">
```
Можно упростить классы вот так